### PR TITLE
Bug 2002559: User preference for topology list view does not follow when a new namespace is created

### PR DESCRIPTION
**Pull request by yozaam**
**Fixes**: 

https://issues.redhat.com/browse/ODC-6330

**Analysis / Root cause**: 

the url had view hardcoded within it, which resulted in always showing the graph view

**Solution Description**: 

removed the hardcoding, now the value is taken automatically from the preferences

**Screen shots / Gifs for design review**: 

userPreferenceHelmTopology

**Unit test coverage report**: 

**Test setup:**

1. Log in to the console
2. Goto the user preference page
3. Create a project from there
4. Set topology preference as a list, perspective preference as Administrator, and YAML as resource method preference
5. Reload the page
6. Went to developer perspective > Add
7. Create a workload ( from **Helm Chart** )
8. Wait for topology to load ( expecting`list` view )

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by yozaam** [comment]
/cc @invincibleJai
may i get a review

File: frontend/packages/helm-plugin/src/utils/helm-utils.ts [hunk 1]
```diff
@@ -175,7 +175,7 @@ export const getOriginRedirectURL = (
 ) => {
   switch (actionOrigin) {
     case HelmActionOrigins.topology:
-      return `/topology/ns/${namespace}?view=graph`;
+      return `/topology/ns/${namespace}`;
     case HelmActionOrigins.list:
```
Comment (invincibleJai) [inline frontend/packages/helm-plugin/src/utils/helm-utils.ts:179]: do we need the case for `HelmActionOrigins.list` now?

cc @rohitkrai03

Comment (rohitkrai03) [reply frontend/packages/helm-plugin/src/utils/helm-utils.ts:179]: Yeah, that's for helm list page.

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**